### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by Vjerem**
mobile version of contact page

### CODE REVIEW

File: src/components/Contact/ContactInfo/style.jsx [hunk 1]
```diff
@@ -34,6 +40,26 @@ const SContactInfo = styled.div`
...
     color: white;
   }
+  @media only screen and (max-width: 1100px) {
+    flex-wrap: wrap;
+
+    section {
+      width:30%
+    }
+    width {
+      width: 10vh;
```
Comment (KeerhemFR) [inline src/components/Contact/ContactInfo/style.jsx]: have to use vw for width instead of vh

**Review by KeerhemFR — CHANGES_REQUESTED** [review]
Saw one little evil, fix it and it should be good

**Review by KeerhemFR — DISMISSED** [review]
See no evil, but will have to be careful with vw and vh and see how it render. Maybe be subject to some modification later depending the outcome.

File: src/components/Contact/ContactInfo/style.jsx [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   section {
     background-color: var(--color);
     display: flex;
     flex-direction: column;
     justify-content: space-evenly;
     align-items: center;
-    width: 22vw;
-    height: 29vh;
+    width: 23vw;
+
```
Comment (Hacene010) [inline src/components/Contact/ContactInfo/style.jsx]: empty line !!!

File: src/components/Contact/ContactInfo/style.jsx [hunk 3]
```diff
@@ -22,10 +22,16 @@ const SContactInfo = styled.div`
   }
```
Comment (Hacene010) [inline src/components/Contact/ContactInfo/style.jsx:21]: empty line

**Review by Hacene010 — CHANGES_REQUESTED** [review]
plz check the empty lines. Your line breaks must be consistent throughout the project

**Review by KeerhemFR — APPROVED** [review]
See no evil

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**